docs/ReactVaporStore.ts

```diff
@@ -1,5 +1,5 @@
-import { createStore, Store } from 'redux';
+import * as Redux from 'redux';
```

Comment:
Something was broken? Since we don't generate a d.ts file for the Doc I don't think this change was required :)